setup.cfg

```
@@ -54,8 +54,7 @@ exclude =
 
 # Autodiff backends
 jax =
-    jax<0.2.28; platform_system!="Windows"
-    jaxlib<0.1.77; platform_system!="Windows"
+    jax[minimum-jaxlib]<0.2.28; platform_system!="Windows"
```

```suggestion
    jax[minimum-jaxlib]<0.2.29; platform_system!="Windows"
```
This would directly take care of #624